src/resources/Fields/FieldsInterfaces.ts

```diff
@@ -24,6 +24,7 @@ export interface FieldModel {
 }
 
 export interface ListFieldsParams {
+    facetsOnly: boolean;

```

Comment:
Shouldn't it be optional?
```suggestion
    facetsOnly?: boolean;

```